### PR TITLE
Fix Linter

### PR DESCRIPTION
**Pull request by mtrinh11**

### CODE REVIEW

File: translation/transport_test.go [hunk 1]
```diff
@@ -133,12 +133,6 @@ func TestDecodeRequestPartnerIDs(t *testing.T) {
 				ctx = bascule.WithAuthentication(ctxTID, auth)
 			}
 
-			if test.tokenType == "" {
-				ctx = ctxTID
-			} else {
-				ctx = bascule.WithAuthentication(ctxTID, auth)
```
Comment (kristinapathak) [inline translation/transport_test.go:139]: Do we not need this?

File: translation/transport.go [hunk 2]
```diff
@@ -279,7 +272,7 @@ func requestAddPayload(m map[string]string, input io.Reader) (p []byte, err erro
 
 	wdmp.Table = table
 
-	payload, err := ioutil.ReadAll(input)
+	payload, _ := ioutil.ReadAll(input)
```
Comment (kristinapathak) [inline translation/transport.go:275]: Why don't we want to check this error?

**Review by kristinapathak — CHANGES_REQUESTED** [review]

File: translation/transport_test.go (hunk 1 above)
Comment (mtrinh11) [reply translation/transport_test.go:139]: It seems to be repeated code from lines 130 - 134, so I took it out

File: translation/transport.go (hunk 2 above)
Comment (mtrinh11) [reply translation/transport.go]: added a check for this. Initially I thought any error would have been addressed in line 281.

**Review by kristinapathak — APPROVED** [review]

**Pull request merged**